hack/.ci/run-e2e-gke.sh

```diff
@@ -23,6 +23,10 @@ fi
 
 SO_DISABLE_NODECONFIG=${SO_DISABLE_NODECONFIG:-false}
 
+SO_SCYLLACLUSTER_NODE_SERVICE_TYPE=${SO_SCYLLACLUSTER_NODE_SERVICE_TYPE:-Headless}
+SO_SCYLLACLUSTER_NODES_BROADCAST_ADDRESS_TYPE=${SO_SCYLLACLUSTER_NODES_BROADCAST_ADDRESS_TYPE:-PodIP}
+SO_SCYLLACLUSTER_CLIENTS_BROADCAST_ADDRESS_TYPE=${SO_SCYLLACLUSTER_CLIENTS_BROADCAST_ADDRESS_TYPE:-PodIP}
```

Comment:
please make these required so we can be sure it takes the values from a presubmit

Reply:
done